pkg/pillar/types/zedmanagertypes.go

```diff
@@ -152,16 +152,16 @@ func (config AppInstanceConfig) VerifyFilename(fileName string) bool {
 
 // Indexed by UUIDandVersion as above
 type AppInstanceStatus struct {
-	UUIDandVersion     UUIDandVersion
-	DisplayName        string
-	DomainName         string // Once booted
-	Activated          bool
-	ActivateInprogress bool     // Needed for cleanup after failure
-	FixedResources     VmConfig // CPU etc
-	StorageStatusList  []StorageStatus
-	EIDList            []EIDStatusDetails
-	OverlayNetworks    []OverlayNetworkStatus
-	UnderlayNetworks   []UnderlayNetworkStatus
+	UUIDandVersion      UUIDandVersion
+	DisplayName         string
+	DomainName          string // Once booted
+	Activated           bool
+	ActivateInprogress  bool     // Needed for cleanup after failure
+	FixedResources      VmConfig // CPU etc
+	VolumeRefStatusList []VolumeRefStatus
```

Comment:
What is the relationship between this array and the VolumeRefStatus maintained by pubsub?
Seems like some code looks at pubsub and others at this array, which can be a source of (latent) bugs.
The old StorageStatus referred to pubsub objects like VolumeStatus through a lookup of a key, but here you have something completely different.